tests/main/remove-auto-connections/task.yaml

```diff
@@ -0,0 +1,43 @@
+summary: Automatic connection gets dropped on refresh
+
+details: |
+      Automatic connection gets removed and not reported by connections if respective plug
+      is no longer present in the refreshed snap.
+
+prepare: |
+  snap install jq
+  snap pack test-snap-v1
```

Comment:
Could you please make the test snap directory name match the snap name. I used this technique in another test, add an extension indicating the version number. `simplesnap.v1` and `simplesnap.v2`. This matches the snap name and is just less surprising for casual readers.

While you are at it, the version numbers of those snaps can be just 1 and 2, for even more consistency.